template.erb

```diff
@@ -1 +1,2 @@
 window.stores = <%= JSON.generate(translations) %>;
+window.Features = { wiki_ed: <%= Features.wiki_ed?.to_json %> };
```

Comment:
This is normally handled in the shared `_head` haml template. I'm not sure what this .erb file is for but I don't think we use it in production. @TheTrio do you remember what this erb file is for? Something needed for Hot Module Reloading from https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/5354 ? Or maybe it's just an accident and not used for anything?